geth/jail/execcution_policy.go

```diff
@@ -0,0 +1,196 @@
+package jail
+
+import (
+	"context"
+	"encoding/json"
+	"math/big"
+	"time"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/common"
+)
+
+// ExecutionPolicy provides a central container for the executions of RPCCall requests for both
+// remote/upstream processing and internal node processing.
+type ExecutionPolicy struct{}
```

Comment:
Also, I noticed that we use `manager` and `account` terminology here and in other places, but these both are managers actually :P I think `nodeManager common.NodeManager` and `accountManager common.AccountManager` are longer but easier to reason about.